### PR TITLE
Convert to WebIDL contiguous mode in ReSpec

### PR DESCRIPTION
**Pull request by dontcallmedom**
Additional minor fixes in the process

### CODE REVIEW

**Comment by miguelao** [comment]
Merging in, sorry for the delay.

File: MediaRecorder.html [hunk 1]
```diff
@@ -42,57 +42,29 @@
...
...lence (for audio) and black frames (for video) when the input MediaStream is muted.  When the MediaRecorder is created, the UA <em title="must" class="rfc2119">must</em> set this attribute to <code>false</code>.</dd><dt><dfn><code>videoBitsPerSecond</code></dfn> of type <span class="idlAttrType"><a>unsigned long</a></span>, readonly       </dt><dd>The value of the Video encoding target bit rate that was passed to the Platform (potentially truncated, rounded, etc), or the calculated one if the user has specified <code>bitsPerSecond</code>.</dd><dt><dfn><code>audioBitsPerSecond</code></dfn> of type <span class="idlAttrType"><a>unsigned long</a></span>, readonly       </dt><dd>The value of the Audio encoding target bit rate that was passed to the Platform (potentially truncated, rounded, etc), or the calculated one if the user has specified <code>bitsPerSecond</code>.</dd></dl></section><section><h2>Methods</h2><dl data-link-for="MediaRecorder" data-dfn-for="MediaRecorder" class="methods"><dt><dfn><code>start</code></dfn></dt><dd>When a <code>MediaRecorder</code> object’s <code>start()</code> method is invoked, the UA <em title="must" class="rfc2119">must</em> run the following steps:
```
Comment (yell0wd0g) [inline MediaRecorder.html:67]: Actually, can we still reflow these paragraphs in a non-minified way?
Like:

<dl blahblah>
  <dt>
  </dt>
</dl>

with LFs in between? It's so much easier to edit as text, and that in turn
enables better git-diff reviews etc.

Comment (dontcallmedom) [inline MediaRecorder.html:67]: sure - but that can hopefully happen separately :)

Comment (yell0wd0g) [inline MediaRecorder.html:67]: Ok, would you do the LF reintroduction, or should we add an issue?
Either way LGTM :)

**Pull request merged**